_opensearch/rest-api/explain.md

```diff
@@ -0,0 +1,163 @@
+---
+layout: default
+title: Explain
+parent: REST API reference
+nav_order: 140
+---
+
+# Explain
+Introduced 1.0
+{: .label .label-purple }
+
+Wondering why a specific document ranks higher (or lower) for a query? You can use the explain API for an explanation of how the relevance score (`_score`) is calculated for every result.
+
+OpenSearch uses a probabilistic ranking framework called [Okapi BM25](https://en.wikipedia.org/wiki/Okapi_BM25) to calculate relevance scores. Okapi BM25 is based on the original [TF/IDF](http://lucene.apache.org/core/4_0_0/core/org/apache/lucene/search/package-summary.html#scoring) framework used by Apache Lucene.
+
+The explain API is an expensive operation in terms of both resources and time. We recommend you don't use it for production workloads.
+{: .warning }
+
+
+## Example
+
+To see the explain output for all results, set the `explain` flag to `true` either in the URL or in the body of the request:
+
+```json
+POST kibana_sample_data_ecommerce/_search?explain=true
+{
+  "query": {
+    "match": {
+      "customer_first_name": "Mary"
+    }
+  }
+}
+```
+
+To see the explain output for a specific document, specify the document ID in the URL:
```

Comment:
This seems like the more common use case. Something like, "More often, you want the output for a single document. In that case, specify the document ID in the URL::